testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -436,7 +437,9 @@
 end
 
 Then(/^I should be logged in$/) do
-  raise 'User is not logged in' unless all(:xpath, "//a[@href='/rhn/Logout.do']").any?
+  repeat_until_timeout(message: 'User is not logged in') do
```

Comment:
I understand why this is necessary, so LGTM. What is expected delay?

Reply:
After a tomcat start it is mostly 3-5 seconds. Later it is less because tomcat has cached the page already.

Reply:
OK, thanks :-) I think this is totally reasonable time if it is expected to resolve problem we have with logging on head.